jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -529,8 +529,78 @@ public StringBuffer append (float value) {
  * @param		value	the integer
  * @return		this StringBuffer
  */
-public StringBuffer append (int value) {
+public synchronized StringBuffer append(int value) {
+	/*[IF Sidecar19-SE]*/
+	/*[IF Sidecar19-SE-B174]*/
+	int currentLength = lengthInternalUnsynchronized();
+	int currentCapacity = capacityInternal();
+
+	int valueLength = Integer.stringSize(value);
+
+	int newLength = currentLength + valueLength;
+
+	if (newLength > currentCapacity) {
+		ensureCapacityImpl(newLength);
+	}
+
+	if (String.enableCompression) {
+		if (count >= 0) {
+			Integer.getChars(value, newLength, this.value);
+
```

Comment:
Nitpick: Can you delete the extra blank lines between the getChars and the count assignment?  Here and in the other copies as well?

Reply:
Fixed in de533af